Makefile

```diff
@@ -949,7 +949,7 @@ editorconfig-checker:
 
 .PHONY: docker
 docker:
-	docker build --disable-content-trust=false -t $(DOCKER_REF) .
+	docker build --disable-content-trust=false --platform linux/amd64 -t $(DOCKER_REF) .
```

Comment:
> `--platform linux/amd64`

Should we export this as `DOCKER_DEFAULT_PLATFORM` in the environment, or does this run fine as an AMI for deployment?